kernel-util/src/main/java/org/sakaiproject/util/Validator.java

```diff
@@ -359,7 +359,27 @@ public static String limitFormattedText(String value, int length)
         return ret.toString();
     }
 
- 
+	/**
+	Checks whether the email address has got valid local domain
+	@param email
+	 */
+	public static boolean isValidDomain(String email) {
```

Comment:
Can we have it like isAllowedLocalEmailDomain as it checks for official local domains